### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by AhmadFattahi**
I just added a test line in the beginning to see how the branching process works.

### CODE REVIEW

**Comment by AhmadFattahi** [comment]
I am just testing a pull process. I added a trivial test comment to one of your source files.

**Comment by JeromeLefebvre** [comment]
That is a fine request. I'll merge it and close the pull-request.

**Comment by AhmadFattahi** [comment]
Thanks Jerome. I just deleted my branch so everything is back in your repo. Is this area where people can talk about code lines?

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+## Ahmad's test ##
```
Comment (AhmadFattahi) [inline .gitattributes:1]: Let's discuss.

Comment (AhmadFattahi) [inline .gitattributes]: And further discuss.

Comment (JeromeLefebvre) [inline .gitattributes:1]: I see you made a change on this line.

**Comment by JeromeLefebvre** [comment]
Deleting your branch has no impact on how my branch will work. They have the same origin, but they have no knowledge of each other.

I'll go and undo this last commit.

**Comment by JeromeLefebvre** [comment]
Like I said, I forgot how to undo a commit. so I did it according to:
http://stackoverflow.com/questions/448919/how-can-i-remove-a-commit-on-github

File: .gitattributes (hunk 1 above)
Comment (JeromeLefebvre) [inline .gitattributes:1]: Sorry that I had to delete this line.

Comment (AhmadFattahi) [inline .gitattributes:1]: Sure thanks for doing this exercise with me, Jerome!

Best regards,
Ahmad Fattahi, PhD | Group Leader, Development Support <vcampus.osisoft.com> | OSIsoft, LLC, 777 Davis St, San Leandro, CA

**Pull request merged**